corehq/apps/sms/static/sms/js/sms_usage_report.js

```diff
@@ -0,0 +1,15 @@
+hqDefine("sms/js/sms_usage_report",[
+    'jquery',
+    'knockout',
+    'hqwebapp/js/daterangepicker.config', //createDateRangePicker
+], function (
+    $
+) {
+    var labels = {
```

Comment:
Would using `createBootstrap3DefaultDateRangePicker` work here so that you didn't need to create the labels?

If you do keep this code, the labels should be translated: `'Last 7 Days'` => `gettext('Last 7 Days')`

Reply:
Used `createBootstrap3DefaultDateRangePicker`.
Thanks for the suggestion 😄